tests/doc_test/doc_needextract/subfolder/file_1.rst

```diff
@@ -0,0 +1,65 @@
+subfolder file
+==============
+
+.. story:: Subfolder example with an absolute image
+   :id: US_SUB_001
+
+   Use an absolut path for the image
+
+   .. image:: /_images/smile.png
+
+   Path: ``/_images/smile.png``
+
+.. story:: Subfolder example with a relative image
+   :id: US_SUB_002
+
+   Use a relative path for the image
+
+   .. image:: ../_images/smile.png
+
+   Path: ``../_images/smile.png``
+
+.. story:: Subfolder example with image in same folder (indirect relative path)
+   :id: US_SUB_003
+
+   Use a relative path for the image
+
+   .. image:: ../subfolder/smile.png
+
+   Path: ``../subfolder/smile.png``
+
+.. story:: Subfolder example with image in same folder (direct relative path)
+   :id: US_SUB_004
+
+   Use a relative path for the image
+
+   .. image:: smile.png
+
+   Path: ``smile.png``
+
```

Comment:
I think a testcase using absolute paths would be good for completeness. In addition, the user story would need to be extracted in both `index.rst` and `file_1.rst`.

The intention is to test absolute paths outside of `_images`. Image could also be in a new folder, but I am not sure if that would be necessary.

```suggestion
.. story:: Subfolder example with image in same folder (indirect absolute path)
   :id: US_SUB_005

   Use a relative path for the image

   .. image:: /subfolder/smile.png

   Path: ``/subfolder/smile.png``

```

Reply:
Thanks, indeed this was missing. :+1: 
Added it.